substrate/frame/bags-list/src/lib.rs

```diff
@@ -341,9 +395,17 @@ pub mod pallet {
 			heavier: AccountIdLookupOf<T>,
 			lighter: AccountIdLookupOf<T>,
 		) -> DispatchResult {
+			ensure!(LockOrder::<T, I>::get().is_none(), Error::<T, I>::ReorderingLocked);
+
 			let _ = ensure_signed(origin)?;
 			let lighter = T::Lookup::lookup(lighter)?;
 			let heavier = T::Lookup::lookup(heavier)?;
+
+			match (Self::remove_if_zero(&heavier), Self::remove_if_zero(&lighter)) {
+				(Ok(true), _) | (_, Ok(true)) => return Ok(()),
+				_ => (),
+			};
```

Comment:
If one of the nodes are not found, we should return error anyways. This should be fine right?
```suggestion
			if Self::remove_if_zero(&heavier)? || Self::remove_if_zero(&lighter)? {
			    // one or both of the nodes had zero score and removed. Nothing to order.
				return Ok(())
			};
```